test.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -x
+
+export AVAR="kaludios"
+envsubst '${AVAR}' < machine.sh
+
```

Comment:
Is this file just for testing?

Reply:
yes, sorry commited it by mistake, i have removed this from the PR #980